docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -159,63 +130,62 @@ Follow these steps:
    git submodule update --init --recursive
    ```
 
-4. Build PyTorch for ROCm.
+4. Build PyTorch for ROCm. The default build directory is `rocm/pytorch:latest-base`.
```

Comment:
@LisaDelaney This doesn't sound right: "The default build directory is `rocm/pytorch:latest-base`". That's the docker image tag, not the build directory.